module_4/src/db/index.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+import fs from 'fs';
+import Sequelize from 'sequelize';
+import path from 'path';
+import { fileURLToPath } from 'url';
+import generalConfig from '../../config/dbConfig.json';
+
+class DB {
+  constructor() {
+    this.__filename = fileURLToPath(import.meta.url);
```

Comment:
is it possible to use default node's capabilities: `__filename` or `__dirname` ?

Reply:
As I understand both __filename and __dirname, in the case of using ES modules, need to be created through import.meta.url. 
https://nodejs.org/api/esm.html#esm_no_filename_or_dirname
a lot of examples follow this way.